### ISSUE TITLE
Jumping to anchors in tabs

### ISSUE THREAD

**Issue by JosiahCarberry** [pojome/elementor#2292]
If an anchor is created in a tab widget, other than in the first, default tab, when a URL is entered in a browser specifying that anchor, the tab with the anchor is not displayed. Instead, the page is displayed showing the first, default tab of the widget.

For example, create a page mypage with a tab widget having 3 tabs. Embed the code <a id="myanchor"></a> within the <div> that corresponds to the second tab on that page. If you enter the URL http://mysite.com/mypage/#myanchor the browser does not display the page with the second tab open.

**Comment by arielk** [comment]
I merged this issue with another issue #1264 in the same subject so we could deal with it more efficiently. So please let's continue the conversation there.

Thanks for your feedback!

**Comment by ShadowCrownServices** [comment]
While I appreciate this has been closed, it can still be found through the above searched problem, as I was facing and playing with the same issue.

Not certain if the above coding eventually went into Elementors core coding... if it has, now and without creating reams of additional page scripting.

I did a page inspect, identified my tab section and the tab IDs:
div class="elementor-tabs-wrapper">
div id="elementor-tab-title-2021" class="elementor-tab-title elementor-tab-desktop-title elementor-active" data-tab="1" role="tab" aria-controls="elementor-tab-content-2021">Example/div>
div id="elementor-tab-title-2022" class="elementor-tab-title elementor-tab-desktop-title elementor-active" data-tab="2" role="tab" aria-controls="elementor-tab-content-2022">Example1
div id="elementor-tab-title-2023" class="elementor-tab-title elementor-tab-desktop-title elementor-active" data-tab="3" role="tab" aria-controls="elementor-tab-content-2022">MyDocument
/div>'

Pulled out:
id="elementor-tab-title-2023"

Then within tab 1 [Example] i placed a link connecting it to a document in another tab :
`domianname.co.uk/page/#elementor-tab-title-2023'

Bingo, works first time every time.
